css/css-values/support/mixed-units-11.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+        font-size: 16px;
+      }
+      .box {
+        background: rgb(0, 0, 255);
+        width: 100%;
+        height: 10px;
+      }
+      @media (width: calc(8000vh * 1vw / 1em * 8px / .4vh)) {
```

Comment:
If my math is right,

```
8000vh = 8000 * 10px / 100 = 800px
1vw = 100px / 100 = 1px
1em = 16px
.4vh = 0.4 * 10px / 100 = 0.04px
```

But then `800 * 1 / 16 * 8 / 0.04` is `10000`. Use `40vh` instead of `.4vh`?

Reply:
Ah yes, I think I was multiplying by `.4` in my head
```
8000vh * 1vw /  1em  * 8px / 40vh
800px  * 1px / 16px  * 8px /  4px
800px²       / 16px  * 8px /  4px
50px                 * 8px /  4px
400px                      /  4px
100px
```

Reply:
Resolved in [6753749](https://github.com/web-platform-tests/wpt/pull/42517/commits/6753749df4f8a9f78c67b5256548fd01caa220e9)